flink-connectors/flink-sql-connector-elasticsearch6/pom.xml

```diff
@@ -40,6 +40,12 @@ under the License.
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-connector-elasticsearch6</artifactId>
 			<version>${project.version}</version>
+			<exclusions>
+				<exclusion>
+					<groupId>org.apache.flink</groupId>
+					<artifactId>flink-connector-base</artifactId>
```

Comment:
should be unncessary because there are no remaining transitive Flink dependencies after shading